aider.el

```diff
@@ -381,6 +402,90 @@ The command will be formatted as \"/ask \" followed by the text from the selecte
   :lighter " Aider"
   :keymap aider-minor-mode-map)
 
+(defvar aider-prog-mode-map (make-sparse-keymap)
+  "Keymap for Aider commands in programming modes.")
+
+;;;###autoload
+(define-minor-mode aider-prog-mode
+  "Minor mode for Aider programming mode integration."
+  :lighter " Aider"
+  :keymap aider-prog-mode-map)
+
+;;;###autoload
+(defun aider-setup-doom-bindings ()
+  "Set up Doom-specific keybindings for Aider in programming modes."
+  (when (and (featurep 'evil) (fboundp 'map!))
+    ;; First declare the prefix
+    (map! :map nil
+          :leader
+          (:prefix ("l" . "aider")))
+
+    ;; Root level commands
+    (map! :map aider-prog-mode-map
+          :leader
+          (:prefix ("l" . "aider")
+                   ))
+
+    ;; Submenus
+    (map! :map aider-prog-mode-map
```

Comment:
This menu looks useful for Doom emacs user. Do you mind attach a screenshot for this? Thanks

Reply:
yeah you bet!

Reply:
Thanks